### DISCUSSION TITLE
How to support jaeger in Multi-cluster environment?

### DISCUSSION THREAD

**Discussion by penghuazhou** [kiali/kiali#4584] [General]
I have a test environment and a production environment, and both environments report data to the same Jaeger. How does Kiali in production only show jaeger data in production?

**Comment by lucasponce** [comment]
Hi @penghuazhou, I moved the issue into a discussion as it's more a questions rather than something actionable (we can move it back later).

Kiali connects with a Jaeger to read information, so if the Jaeger used is collecting information from several data planes, it should work without special changes.

A different question is if inside Jaeger you have explicit clusterId "tenants" that would be needed to created some aggregated view, then perhaps we are talking about a different issue.

In short, pointing Kiali to the Jaeger used would be enough, if there is any problem I'd like to get more details as we are actively working on this topic to improve the integration with Jaeger as well.
